SB_Flowchart.yaml

```diff
@@ -0,0 +1,448 @@
+    text: >-
+      Is the segment located at or near the end of the video?
+    answers:
+      - text: "Yes"
+        target: result
+        target_id: outro
+      - text: "No"
+        target: result
+        target_id: intermission
+
+  do_switch_platform_exclusive_rapidly:  # 15
+    text: >-
+      Do they switch between promoting the platform and their exclusives rapidly?
+    answers:
+      - text: "Yes"
+        target: result
+        target_id: sponsor
+      - text: "No"
+        target: result
+        target_id: platform_split_exclusive
+
+  segment_contain_animation_or_frame:  # 16
+    text: >-
+      Does this segment contain an animation or still frame?
+      (May contain audio)
+    answers:
+      - text: "Yes"
+        target: question
+        target_id: is_segment_in_other_videos_same_creator
+      - text: "No"
+        target: question
+        target_id: are_promoting_something
+
+  is_segment_in_other_videos_same_creator:  # 17
+    text: >-
+      Is this segment similar to other segments in videos by the same creator?
+    answers:
+      - text: "Yes"
+        target: question
+        target_id: is_disclaimer_to_protect
+      - text: "No"
+        target: result
+        target_id: nothing
+
+  has_creator_exclusives_on_platform:  # 20
+    text: >-
+      Does the creator have exclusive videos on the platform?
+    answers:
+      - text: "Yes"
+        target: question
+        target_id: do_switch_platform_exclusive_rapidly
+      - text: "No"
+        target: result
+        target_id: sponsor
+
+  are_promoting_something:  # 21
+    text: >-
+      Are they promoting something?
+    answers:
+      - text: "Yes"
+        target: question
+        target_id: is_video_primarly_sponsor_or_has_bias
+      - text: "No"
+        target: question
+        target_id: asking_to_like_subscribe_comment_share
+
+  is_video_primarly_sponsor_or_has_bias:  # 22
+    text: >-
+      Is the video primarily about the sponsor/ merch or could this cause a bias?
+    answers:
+      - text: "Yes"
+        target: result
+        target_id: not_disclosure_consider_full_video
+      - text: "No"
+        target: question
+        target_id: is_sponsor_in_this_segment
+
+  is_sponsor_in_this_segment:  # 23
+    text: >-
+      Is there a _sponsor_ in this segment?
+      (See _sponsor_ bubble)
+    answers:
+      - text: "Yes"
+        target: question
+        target_id: is_obvious_sponsor_owned_by_creator
+      - text: "No"
+        target: question
+        target_id: is_related_to_creator
+
+  is_obvious_sponsor_owned_by_creator:  # 24
+    text: >-
+      Is it obvious the sponsor is owned by the same company/person?
+    answers:
+      - text: "Yes"
+        target: result
+        target_id: unpaid
+      - text: "No"
+        target: question
+        target_id: is_sponsor_nebula_or_curiosity
+
+  is_sponsor_nebula_or_curiosity:  # 25
+    text: >-
+      Is it obvious the sponsor is owned by the same company/person?
```

Comment:
Wrong text.
Should be: `Is the sponsor Nebula or Curiosity Stream?`

Reply:
Fixed it, must be a copying issue. I force pushed the new commits